test/WebSocketsTestApp/runtimeconfig.template.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
We're open source as well, don't forget that. We need people running public bits and private bits so we understand what contributors can/can't do. I'd leave this for now because it's harmless and it lets RC4 work.